Formula/pipe-rename.rb

```diff
@@ -0,0 +1,18 @@
+class PipeRename < Formula
+  desc "Rename your files using your favorite text editor"
+  homepage "https://github.com/marcusbuffett/pipe-rename"
+  url "https://crates.io/api/v1/crates/pipe-rename/1.4.2/download"
```

Comment:
Can we use a GitHub tarball?

Reply:
There are no github releases and the last github tag is version 1.1.6 from September 2020.